pkg/controller/services.go

```diff
@@ -984,6 +984,8 @@ func (cont *AciController) opflexDeviceChanged(obj apicapi.ApicObject) {
 
 		for _, node := range nodeUpdates {
 			cont.env.NodeServiceChanged(node)
+			cont.erspanFullSync()
```

Comment:
Why would this be called multiple times in a loop?

Reply:
You are right, this is not needed. I have called the erspanFullSync in environment.go